erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md

```diff
@@ -0,0 +1,102 @@
+<!-- add-breadcrumbs -->
+
+# Process Statement Of Accounts
+
+> Introduced in Version 13
+
+**Process Statement Of Accounts is a tool which helps you send Statement Of Account (General Ledger Report) and Ageing (Accounts Receivable Summary) Report as a PDF to your customers in bulk via email either manually or automated periodically.**
+
+<br>
+To access *Process Statement Of Accounts* list you can search "Process Statement of Accounts" in the navbar.
+
+## 1. Prerequisites
+
+1. The tool needs to use the email-ids of customers to send them the reports. On not finding the below email entries in the Customer contacts the tool won't allow you to select the respective customer, so please make sure the following details are filled in the customer documents.
+
+    - Billing Email of Customer: This is mandatory and can be set within the [customer contact](/docs/user/manual/en/CRM/contact#1-how-to-create-a-contact) with "Is Billing Contact" option checked.
+    - Primary Email of Customer: This is not mandatory, unless you select the "Send To Primary Contact" in the form.
+
+2. Email Account setup with outgoing enabled. Learn more about this [here](/docs/user/manual/en/setting-up/email/email-account).
+
+
+## 2. How to create a Process Statement Account entry
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/name_and_filters.png">
+
+
+1. Go to the "Process Statement Of Account" list view by searching in the navbar and click on "New".
+
+2. Enter a name for the entry, for future reference.
+
+3. Set the General Ledger filters for the statements which will be sent to the customers.
+
+    - "From Date" and "To Date" filters will be hidden and auto-filled dynamically when "Enable Auto Email" option is selected.
+    - "Project" and "Cost Center" fields are multi-select.
```

Comment:
what does multi select mean? You're not talking to your dev buddies

Reply:
https://docs.erpnext.com/docs/user/manual/en/customize-erpnext/articles/table-multiselect-field.md

I'll link this so people get familiar with the field type.